src/geode/geosciences/representation/io/structural_model_input.cpp

```diff
@@ -42,7 +42,9 @@ namespace geode
                 structural_model.nb_lines(), " Lines, ",
                 structural_model.nb_corners(), " Corners, ",
                 structural_model.nb_faults(), " Faults and ",
-                structural_model.nb_horizons(), " Horizons" );
+                structural_model.nb_horizons(), " Horizons and ",
```

Comment:
If there is no fault (or something else), should we disply "0", "none" or do not display anything? Is a message like "0 faults" relevant?

Reply:
Good question! I will say no message if nothing. What is your opinion?